src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenClosureLambdaTests.cs

```diff
@@ -3243,9 +3243,24 @@ static void Main()
         Func<string> f = x.ToString;
         Console.Write(f.Invoke());
     }
-}"
-;
-            CompileAndVerify(source, expectedOutput: "42");
+}";
+            var verifier = CompileAndVerify(source, expectedOutput: "42",
+                verify: Verification.FailsILVerify with { ILVerifyMessage = "[Main]: Unrecognized arguments for delegate .ctor. { Offset = 0xe }" });
```

Comment:
Filed issue [https://github.com/dotnet/runtime/issues/96695](https://github.com/dotnet/runtime/issues/96695) to confirm with runtime team whether the problem is with ILVerify or Roslyn

Reply:
Consider tagging the issue in source so readers have context.